tests/test_drawImage.ipynb

```diff
@@ -0,0 +1,85 @@
xpJlHNlQGuVxaE3+tk8Av2R7PfA9wAclrQduAx61vQ54tPweEW1gN1uGTG1Cs33Q9pPl59eA54BLgU3A/eVm9wPvXaggI2Jxyc2WYXNGbWiSLgfeCewGVts+WBZ9haJKGhGj7mwYWCvpfODTwIdsvyrp/8tsW5o9X0vaAmwBWMl584s2IhbFqHYKNGoxlrSMIpl9wvZnytWHJK0py9cAh2fb1/Y222O2x5axoh8xR8QC01SzZdjUJjQVt2L3Ac/ZvrujaAdwS/n5FuBz/Q8vIhadGdlOgSZVzu8Ffgp4RtJT5boPAx8BHpZ0K/Bl4KaFCTFqVf3FqhnyETGbYWzwb6I2odn+B4qhKbO5rr/hRMRQaGtCi4izyygPrE1Ci4jp7EzwGBEtMpr5LAktImZKlTMi2sHAiFY5BzwVQ0QMJTdcGpC0UdLzksYlzZjEQtIKSQ+V5bvLRyyR9G5JT0h6pvzzh+rOlYQWETP06+F0SUuAe4HrgfXA5nK2nk63AkdtXwncA9xVrj8C/Ljtd1AM3n+g7nxJaBExg6bcaGlgAzBu+0XbJ4EHKWbq6dQ5c88jwHWSZPtfbP93uX4vcK6kyucnk9AiYrqm1c1mVc5LgZc6vu8v1826je0J4Bhwcdc2Pwk8abvyrdvpFIiIaYqBtY07BS6RtKfj+zbb2/oaj/Q2imroe+q2TUKLiJmaz6RxxPZYRfkBYG3H98vKdbNts1/SUuAC4GUASZcBnwV+2vZ/1AWTKmdEzCC70dLA48A6SVdIWg7cTDFTT6fOmXtuAB4r51h8M/CXwG22/7HJyZLQImK6PrahlW1iW4FdFNP3P2x7r6Q7Jf1Eudl9wMWSxoFf5OvvJ9kKXAncLumpcvmmqvOlyhkRXfr7LKftncDOrnW3d3w+Dtw4y36/CfzmmZwrCS0iZhrCyRubSEKLiOna/KLhiDgL5Q4tIlpjNPNZElpEzKSp0axzJqFFxHTmTAbWDpUktIiYRjQeNDt0ktAiYqYktIhojSS0iGiFtKFFRJuklzMiWsKpckZES5gktIhokdGscSahRcRMGYcWEe0xogmtdsZaSWsl/Z2kZyXtlfTz5fo7JB3omEnyRxc+3IhYcDZMTjVbhkyTO7QJ4JdsPynpG4AnJP1tWXaP7d9ZuPAiYiBG9A6tNqHZPggcLD+/Juk5Zr5XLyLaZEQT2hm9JEXS5cA7gd3lqq2Snpa0XdKFPfbZImmPpD2nqHxHaEQMAwNTbrYMmcYJTdL5wKeBD9l+FfgY8FbgKoo7uI/Otp/tbbbHbI8to/It7hExFAyearYMmUa9nJKWUSSzT9j+DIDtQx3lHwf+YkEijIjFZYaywb+JJr2conhv3nO27+5Yv6Zjs/cBX+p/eBExEHazZcg0uUP7XuCngGckPVWu+zCwWdJVFPl8H/CBBYkwIhbfECarJpr0cv4DoFmKds6yLiJG3nDefTWRJwUiYjoDmT4oIlojd2gR0Q4e2V7OJLSImM7gIRxj1kQSWkTMNIRPATSRhBYRM6UNLSJawU4vZ0S0SO7QIqIdjCcnBx3EnCShRcR0p6cPGkFnNB9aRJwl+jh9kKSNkp6XNC7ptlnKV0h6qCzfXc67eLrs18r1z0v6kbpzJaFFxDQGPOVGSx1JS4B7geuB9RSTWqzv2uxW4KjtK4F7gLvKfdcDNwNvAzYCf1Aer6cktIiYzn2d4HEDMG77RdsngQeBTV3bbALuLz8/AlxXTlu2CXjQ9gnb/wmMl8frKW1oETFDHzsFLgVe6vi+H7i61za2JyQdAy4u1/9T176V7zNZ1IT2GkePfN6PfLlj1SXAkcWM4QwMa2zDGhcktrnqZ2zfMt8DvMbRXZ/3I5c03HylpD0d37fZ3jbfGOZqUROa7W/s/C5pj+2xxYyhqWGNbVjjgsQ2V8MWm+2NfTzcAWBtx/fLynWzbbNf0lLgAuDlhvtOkza0iFhIjwPrJF0haTlFI/+Orm12ALeUn28AHrPtcv3NZS/oFcA64J+rTpY2tIhYMGWb2FZgF7AE2G57r6Q7gT22d1C8s+QBSePAKxRJj3K7h4FnKV54/kHblY17g05oA6trNzCssQ1rXJDY5mqYY5s32zvpmrLf9u0dn48DN/bY97eA32p6LnlEn9mKiOiWNrSIaI2BJLS6RyEGSdI+Sc9IeqqrO3oQsWyXdFjSlzrWXSTpbyW9UP554RDFdoekA+W1e0rSjw4otrWS/k7Ss5L2Svr5cv1Ar11FXENx3dpg0auc5aML/w68m2Kg3OPAZtvPLmogPUjaB4zZHviYJUnfD7wO/Kntt5frfht4xfZHyv8MLrT9q0MS2x3A67Z/Z7Hj6YptDbDG9pOSvgF4Angv8DMM8NpVxHUTQ3Dd2mAQd2hNHoUIwPYXKHp9OnU+JnI/xT+IRdcjtqFg+6DtJ8vPrwHPUYwwH+i1q4gr+mQQCW22RyGG6Zdq4G8kPSFpy6CDmcVq2wfLz18BVg8ymFlslfR0WSUdSHW4UzlzwzuB3QzRteuKC4bsuo2qdArMdI3td1HMDvDBsmo1lMrBh8PUTf0x4K3AVcBB4KODDEbS+cCngQ/ZfrWzbJDXbpa4huq6jbJBJLQzfpxhMdk+UP55GPgsNU/3D8Chsi3mdJvM4QHH8/9sH7I96eIdaB9ngNdO0jKKpPEJ258pVw/82s0W1zBdt1E3iITW5FGIgZC0qmysRdIq4D3Al6r3WnSdj4ncAnxugLFMczpZlN7HgK5dOfXMfcBztu/uKBrotesV17BctzYYyMDaslv6d/n6oxCNRwIvJEnfSnFXBsVTFJ8cZGySPgVcSzEbwyHg14E/Bx4G3gJ8GbjJ9qI3zveI7VqKapOBfcAHOtqsFjO2a4C/B54BTk/a9WGK9qqBXbuKuDYzBNetDfKkQES0RjoFIqI1ktAiojWS0CKiNZLQIqI1ktAiojWS0CKiNZLQIqI1ktAiojX+D++mDQLL0NMPAAAAAElFTkSuQmCC\n",
```

Comment:
Looks like this notebook needs its outputs clearing...

Reply:
Deleted notebook (replaced with `test_HSM.ipynb`).